Add documentation to v2 (#144)
microsoft/OHDSIonAzure: README.md

* [Achilles](https://github.com/OHDSI/Achilles) - provides descriptive statistics on an OMOP CDM database
* [ETL-Synthea](https://github.com/OHDSI/ETL-Synthea) - Conversion from Synthea CSV to OMOP CDM

## Overview

![alt text](./docs/media/OHDSIonAzure.png "Architecture")

Once deployed, The OHDSI on Azure solution will have the following components:

> 1. Your Atlas and Webapi instances will be deployed as container on Azure App Service. The images get pulled from Docker Hub. Both app services will be deployed in a single App Service Plan.

Reviewer: ```suggestion
1. An Atlas and Webapi instances will be deployed as container on Azure App Service. The images get pulled from Docker Hub. Both sites will be deployed into a single App Service Plan.
```

2. A Storage Account will be deployed with a filesystem. This is used to store the Atlas configuration files.
3. Azure database for PostgreSQL will be deployed with different databases - one for the OMOP CDM and one for the Atlas WebAPI schemas.
4. Key Vault will be deployed to store the database credentials and other secrets.

> 5. Log Analytics workspace will be deployed to collect logs from the App Services, PostgreSQL and Key Vault.

Reviewer: All items on this list are deployed as mentioned in the line before the list. I think you can remove it from all items here.

Author: Deleted

> * You can host your CDM in Azure PostgreSQL. You can load your vocabularies into Azure Storage Container as cs.gz files, and pass as a paramater in your custom deployment.

Author: Fixed
